tools/harness-thci/OpenThread_WpanCtl.py

```diff
@@ -88,22 +71,24 @@ def __init__(self, **kwargs):
             self.handle = None
             self.AutoDUTEnable = False
             self._is_net = False  # whether device is through ser2net
-            self.logStatus = {
-                'stop': 'stop',
-                'running': 'running',
-                'pauseReq': 'pauseReq',
-                'paused': 'paused',
-            }
+            self.logStatus = {'stop': 'stop', 'running': 'running', 'pauseReq': 'pauseReq', 'paused': 'paused'}
             self.logThreadStatus = self.logStatus['stop']
-            self.connectType = (
-                (kwargs.get('Param5')).strip().lower()
-                if kwargs.get('Param5') is not None
-                else 'usb'
+            self.connectType = (kwargs.get('Param5')).strip().lower() if kwargs.get('Param5') is not None else 'usb'
+            (self.prompt, self.wpan_cmd_prefix, self.wpan_interface) = (
+                kwargs.get('Param8').strip().split(',') if kwargs.get('Param8') else ['#', 'wpanctl', 'wpan0']
```

Comment:
nit: parameters like `Param8`, `Param5` are not very self-explanatory

Reply:
These are extra parameters for TH analyzing deviceInputFields.xml.